flink-table/flink-table-common/src/main/java/org/apache/flink/table/module/CoreModule.java

```diff
@@ -29,19 +32,28 @@
 /** Module of default core metadata in Flink. */
 public class CoreModule implements Module {
     public static final CoreModule INSTANCE = new CoreModule();
+    private List<BuiltInFunctionDefinition> builtInFunctionDefinitions;
```

Comment:
can be `final`.

Reply:
Thanks for your review，I have revised the above two.